generator/src/main.rs

```diff
@@ -31,48 +31,77 @@ impl FromStr for Mode {
     }
 }
 
-#[derive(clap::Parser, Debug)]
-#[clap(author, version, about)]
-/// A transaction generator used for testing performance of the chain.
-struct App {
+#[derive(Debug, Subcommand)]
+enum Command {
+    /// Send CCD to a list of receivers.
+    Ccd(CcdArgs),
+    /// Mint CIS-2 NFT tokens.
+    MintNfts,
+    /// Transfer CIS-2 tokens to a list of receivers.
+    TransferCis2(TransferCis2Args),
+    /// Wrap, unwrap, and transfer WCCD tokens. First, wCCD are minted for every
+    /// account on the chain and then 1 wCCD is alternately wrapped,
+    /// transferred, and unwrapped.
+    Wccd,
+    /// Register Web3 ID credentials.
+    RegisterCredentials,
+}
+
+#[derive(Debug, Args)]
+pub struct CcdArgs {
+    #[arg(long = "receivers", help = "Path to file containing receivers.")]
+    receivers: Option<PathBuf>,
     #[clap(
-        long = "node",
-        help = "GRPC interface of the node.",
-        default_value = "http://localhost:20000"
+        long = "amount",
+        help = "CCD amount to send in each transaction",
+        default_value = "0"
     )]
-    endpoint:  Endpoint,
-    #[clap(long = "sender")]
-    account:   PathBuf,
-    #[clap(long = "receivers")]
-    receivers: Option<PathBuf>,
+    amount:    Amount,
     #[clap(
         long = "mode",
         help = "If set this provides the mode when selecting accounts. It can either be `random` \
                 or a non-negative integer. If it is an integer then the set of receivers is \
                 partitioned based on baker id into the given amount of chunks."
     )]
     mode:      Option<Mode>,
-    #[clap(long = "tps")]
-    tps:       u16,
+}
+
+#[derive(Debug, Args)]
+pub struct TransferCis2Args {
+    #[arg(long = "receivers", help = "Path to file containing receivers.")]
+    receivers: Option<PathBuf>,
+}
+
+#[derive(clap::Parser, Debug)]
+#[clap(author, version, about)]
+/// A transaction generator used for testing performance of the chain.
+struct App {
     #[clap(
-        long = "amount",
-        help = "CCD amount to send in each transaction",
-        default_value = "0"
+        long = "node",
+        help = "GRPC interface of the node.",
+        default_value = "http://localhost:20000"
     )]
-    amount:    Amount,
+    endpoint: Endpoint,
+    #[clap(long = "sender")]
+    account:  PathBuf,
+    #[clap(long = "tps")]
+    tps:      u16,
```

Comment:
Please add help text here. All options should have help text.

![image](https://github.com/Concordium/concordium-misc-tools/assets/105212/c4801cf2-d3e3-4964-9af3-2fcc120b1b54)